python/ray/remote_function.py

```diff
@@ -253,6 +254,37 @@ def _remote(self, args=None, kwargs=None, **task_options):
         max_retries = task_options["max_retries"]
         retry_exceptions = task_options["retry_exceptions"]
 
+        if scheduling_strategy is None or not isinstance(
```

Comment:
we can implement this piece of code in `ray_options_util.py` `_check_deprecate_placement_group` so it can be used by both tasks and actors.

Reply:
Hm, this is not a CHECK() semantics here but warning? Created a separated one for warning so it's shared across task and actor.

Reply:
yea, for us it should be warning instead of throwing the exception.